lib/stopwatch/application.ex

```diff
@@ -13,9 +13,10 @@ defmodule Stopwatch.Application do
       # Start the PubSub system
       {Phoenix.PubSub, name: Stopwatch.PubSub},
       # Start the Endpoint (http/https)
-      StopwatchWeb.Endpoint
+      StopwatchWeb.Endpoint,
```

Comment:
The application will create the timer by calling `Stopwatch.Timer.start_link\1` function